src/modules/twap/utils/isTwapOrderFulfilled.ts

```diff
@@ -0,0 +1,5 @@
+import { TWAPOrderStruct } from '../types'
+
+export function isTwapOrderFulfilled(order: TWAPOrderStruct, executedSellAmount: string): boolean {
+  return executedSellAmount === (BigInt(order.partSellAmount) * BigInt(order.n)).toString()
```

Comment:
Are all orders fillOrKill?
If not, we should allow for small dust amounts not matched due to rounding.

Reply:
For now all part orders are fillOrKill